src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/print_order_statistics.html.twig

```diff
@@ -0,0 +1,42 @@
+<div class="card">
+  <div class="card-body">
+    <div class="row">
+      <div class="col-sm">
+        <p>
+          <strong>{{ 'Date'|trans({}, 'Admin.Global') }}</strong>
+        </p>
+        <strong>{{ orderForViewing.createdAt|date('Y-m-d') }}</strong>
```

Comment:
Shouldn't be localized to the user ?

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17617/commits/880b0ffdaca18f032d2cb941f37e0751b4745479